<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="4" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-18T20:12:13 PKT" name="Suite" finished-at="2024-08-18T20:12:36 PKT" duration-ms="23583">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.invalidOrder()[pri:0, instance:rahulshetty.tests.ErrorValidationTest@595b007d]" name="invalidOrder" class="rahulshetty.tests.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2024-08-18T20:12:13 PKT" name="Submit Order Test" finished-at="2024-08-18T20:12:13 PKT" duration-ms="16">
    </test> <!-- Submit Order Test -->
    <test started-at="2024-08-18T20:12:13 PKT" name="Error Validation Test" finished-at="2024-08-18T20:12:36 PKT" duration-ms="23583">
      <class name="rahulshetty.tests.ErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:rahulshetty.tests.ErrorValidationTest@595b007d]" started-at="2024-08-18T20:12:13 PKT" name="launchApplication" finished-at="2024-08-18T20:12:20 PKT" duration-ms="7725" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="invalidOrder()[pri:0, instance:rahulshetty.tests.ErrorValidationTest@595b007d]" started-at="2024-08-18T20:12:20 PKT" name="invalidOrder" finished-at="2024-08-18T20:12:36 PKT" duration-ms="15779" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidOrder -->
      </class> <!-- rahulshetty.tests.ErrorValidationTest -->
    </test> <!-- Error Validation Test -->
  </suite> <!-- Suite -->
</testng-results>
